### PR TITLE
Feature: Space model updates

### PR DESCRIPTION
**Pull request by brianmuse**
Adds in:
- `state` (submodel)

Removes:
- `next_event`
- `current_event`
- `is_dibsed`

### CODE REVIEW

File: src/main/java/com/robinpowered/sdk/model/Space.java [hunk 1]
```diff
@@ -29,7 +29,6 @@
     // Immutable
     private final int id;
     private final int locationId;
-    private final boolean isDibsed;
```
Comment (Rican7) [inline src/main/java/com/robinpowered/sdk/model/Space.java:32]: RIP

**Comment by christophermark** [comment]
While we're axing dibs, want to deprecate these methods?
https://github.com/robinpowered/robin-java-sdk/blob/e5f22225f4de9e93b127b018966ce4b8afba8a68/src/main/java/com/robinpowered/sdk/service/PlacesService.java#L137-L141

**Comment by brianmuse** [comment]
The dibs model has been deprecated for a while. Just gonna remove entirely.

**Comment by christophermark** [comment]
LGTM :+1:

**Pull request merged**